content/en/docs/tutorials/hello-minikube.md

```diff
@@ -97,6 +97,13 @@ Pod runs a Container based on the provided Docker image.
     kubectl create deployment hello-node --image=k8s.gcr.io/echoserver:1.4
     ```
 
+_Note that the image used in the command above is currently incompatible with Apple Silicon, if the
+pod fails to start with `CrashLoopBackOff` try the command bellow._
+
+  ```shell
+  kubectl create deployment hello-node --image=polyverse/node-echo-server
+  ```
+
```

Comment:
This fix isn't appropriate. Instead, we should fix `k8s.gcr.io/echoserver` to run on AArch64 nodes.